src/common/actions/repositories.js

```diff
@@ -5,8 +5,7 @@ import { conf } from '../helpers/config';
 
 const BASE_URL = conf.get('BASE_URL');
 
-export const FETCH_REPOSITORIES = 'FETCH_REPOSITORIES';
-export const FETCH_REPOSITORIES_SUCCESS = 'FETCH_REPOSITORIES_SUCCESS';
```

Comment:
Shouldn't `FETCH_REPOSITORIES_ERROR` be removed in such case too? As they all can be handled by one action with status and payload?

One thing that worries me in this approach is that by using one action type for all instead of having one `switch` in reducer we need to put much more logic into every case (to handle different statuses of actions).
But maybe it's not such an overhead in small cases like this.